app/views/downloads/_download_preface.html.erb

```diff
@@ -67,17 +67,24 @@
 
     <% related_works = @work.parent_work_relationships.reject { |wr| !wr.parent } %>
     <% if related_works.length > 0 %>
+      <%# i18n-tasks-use t("downloads.download_preface.html.inspired_by.unrevealed")
+          i18n-tasks-use t("downloads.download_preface.html.inspired_by.revealed_html")
+          i18n-tasks-use t("downloads.download_preface.html.translation_of.unrevealed")
+          i18n-tasks-use t("downloads.download_preface.html.translation_of.revealed_html") %>
```

Comment:
I'm assuming that the extra `.html.` here is necessary because of the `.html` in this line:
https://github.com/otwcode/otwarchive/blob/538664f7b1de73539a5c61b28c37c72ca7a0821d/app/views/downloads/show.html.erb#L1
But passing the suffix like that is deprecated, so that line's going to need changing in the 7.0 upgrade. Which means that all of the keys here would have to be renamed as part of the 7.0 upgrade, and this would need to be re-tested. I think it'd be cleaner to fix the deprecated line now, and use the correct key names so that the 7.0 upgrade goes smoother.

Reply:
Happy to resolve it here, thanks for the info!